File: Game Commons/src/de/gymdon/inf1315/game/Castle.java
package de.gymdon.inf1315.game;

public class Castle extends Building {

    public Castle(Player owner, int x, int y) {
	this.owner = owner;
	this.x = x;
	this.y = y;
	this.hp = 10000;
	this.defense = 80;
	this.cost = 0;
	this.income = 100;
    }

    @Override
    public void occupy(Player p) {

    }

    @Override
    public int getSizeX() {
	return 2;
    }

    @Override
    public int getSizeY() {
	return 2;
    }
}

File: Game Commons/src/de/gymdon/inf1315/game/Castle.kt
package de.gymdon.inf1315.game

class Castle(owner: Player?, x: Int, y: Int) : Building() {

    init {
        this.owner = owner
        this.x = x
        this.y = y
        this.hp = 10000
        this.defense = 80
        this.cost = 0
        this.income = 100
    }

    override fun occupy(p: Player) {

    }

    override val sizeX: Int
        get() = 2

    override val sizeY: Int
        get() = 2
}
